src/SwedbankPay.Sdk.Infrastructure/PayeeInfoResponse.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SwedbankPay.Sdk.PaymentOrders;
+using System;
+
+namespace SwedbankPay.Sdk
+{
+    internal class PayeeInfoResponse : Identifiable, IPayeeInfo
+    {
+        public PayeeInfoResponse(PayeeInfoResponseDto dto)
+            :base(dto.Id)
+        {
+            OrderReference = dto.OrderReference;
+            PayeeId = dto.PayeeId;
+            PayeeName = dto.PayeeName;
+            PayeeReference = dto.PayeeReference;
+            ProductCategory = dto.ProductCategory;
+            Subsite = dto.Subsite;
+        }
+
+        public PayeeInfoResponse(PaymentOrderPayeeInfoDto dto)

```

Comment:
Since we only have one `PayeeInfoResponse`, it feels weird to have different DTO classes. I can see that `PaymentOrderPayeeInfoDto` contains a few properties that `PayeeInfoResponseDto` does not have. Should these properties not be exposed in `PayeeInfoResponse`? Perhaps we need a `PaymentOrderPayeeInfo` class as well?

Reply:
I'll double check with the api docs to see if they should be the same.

Reply:
Payments does not expose `CorporationName` and `CorporationId`. Shall I remove them and consolidate the two models?

Reply:
No, separate the response models between payments and paymentorders.